packages/iris-grid/src/IrisGridTableModel.ts

```diff
@@ -296,6 +300,10 @@ class IrisGridTableModel
     );
   }
 
+  get isPartitionAwareSourceTable(): boolean {
+    return this.isPartitionRequired && true;
```

Comment:
I'm dumb, the `&& true` is unnecessary. Just need to check `isPartitionRequired`.
```suggestion
    return this.isPartitionRequired;
```